### PR TITLE
Create ActionBar React component - Closes #541

### LINKED ISSUE #541: Create ActionBar React component
### Expected behaviour
There should be one `ActionBar` component.
It could look for example like this:
```
<ActionBar 
  secondaryButton={{
    label: 'Cancel',
    onClick: someFn,
  }}
  primaryButton= {{
    label: 'Vote',
    onClick: someFn,
    disabled: someExpression,
    className: 'vote-button'
  }}
  fee='1'
  />
```

### Actual behaviour
There are multiple action bars, a lot of code repetition:
https://github.com/LiskHQ/lisk-nano/blob/development/src/components/send/send.js#L109-L117
https://github.com/LiskHQ/lisk-nano/blob/development/src/components/signVerify/signMessage.js#L69-L76
https://github.com/LiskHQ/lisk-nano/blob/development/src/components/passphrase/passphrase.js#L63-L74

And more will come.

### PR DESCRIPTION
**Pull request by slaweet**
Closes #541

### CODE REVIEW

File: src/components/actionBar/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Button from 'react-toolbox/lib/button';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+
+const ActionBar = props => (
+  <section style={{ margin: '0' }}
```
Comment (reyraa) [inline src/components/actionBar/index.js]: Please remove inline style

File: src/components/actionBar/index.test.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import chai, { expect } from 'chai';
+import { mount } from 'enzyme';
+import chaiEnzyme from 'chai-enzyme';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import ActionBar from './';
```
Comment (reyraa) [inline src/components/actionBar/index.test.js]: Please use the name of file

File: src/components/actionBar/index.test.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  });
+
+  it('binds props.secondaryButton.onClick to first button onClick', () => {
+    wrapper.find('Button').at(0).simulate('click');
+    expect(props.secondaryButton.onClick).to.have.been.calledWith();
+  });
+
+  it('binds props.primaryButton.onClick to second button onClick', () => {
+    wrapper.find('Button').at(1).simulate('click');
+    expect(props.primaryButton.onClick).to.have.been.calledWith();
```
Comment (reyraa) [inline src/components/actionBar/index.test.js]: Isn't the primary button disabled?nI was expecting it not to trigger the handler

**Comment by reyraa** [comment]
Thanks Vit, there are just a few changes request.

File: src/components/actionBar/index.test.js (hunk 3 above)
Comment (slaweet) [inline src/components/actionBar/index.test.js]: It is not. See `disabled: false,` on line 26.

Comment (reyraa) [inline src/components/actionBar/index.test.js]: oh sorry. yeah. :-D

**Pull request merged**